samples/app-link-unfurling-in-share-to-teams/csharp/README.md

```diff
@@ -0,0 +1,119 @@
+---
+page_type: sample
+description: This sample demos the feature of link unfurling for Share to teams.
```

Comment:
This sample app shows the feature of link unfurling for Share to teams.

Reply:
Updated